app/validation/validator.py

```diff
@@ -95,12 +96,20 @@ def _validate_blocks(self, json_to_validate, section, group, all_groups, answers
 
             errors.extend(self._validate_placeholders(block))
 
+            errors.extend(self._validate_variants(block, answers_with_parent_ids, numeric_answer_ranges))
+
         return errors
 
     def _validate_questions(self, block, answers_with_parent_ids, numeric_answer_ranges):
```

Comment:
In this content, is block still being referred to as our schema block or just a JSON block because technically its `block_or_variants`

Reply:
I'm not sure if this comment is on the wrong line? _validate_questions takes a block in terms of our schema block.

Reply:
As discussed, referring to the fact that `_validate_questions` are both called on its own with a `block` and inside `_validate_variants` but with variants.

Reply:
:+1: Yup, I misunderstood.